releases/release-1.29/release-notes/maps/pr-118619-map.yaml

```diff
@@ -1,4 +1,4 @@
 pr: 118619
 releasenote:
-  text: 'Dynamic resource allocation: avoid creating a new gRPC connection for every
+  text: 'Dynamic resource allocation: avoided creating a new gRPC connection for every
```

Comment:
text: 'Dynamic resource allocation: will now avoid creating a new `gRPC` connection for every call of prepare/unprepare resource(s).'